README.md

```diff
@@ -18,35 +18,50 @@ these checks verify that the plugin is available for users of napari
 to install, and would register entry points with napari. 
 
 There are two parts to the validation tool. One part is a 
-command line interface and the other is a pytest fixture.
+command line interface, and the other is a pytest fixture.
 
 ### Command Line Interface (CLI) Usage
```

Comment:
Consider adding a link to the CLI usage documentation (documentation/CLI_usage.md) which is super helpful.